src/main/java/org/broadinstitute/hellbender/utils/recalibration/BaseRecalibrationEngine.java

```diff
@@ -67,11 +67,6 @@ public BaseRecalibrationEngine( final RecalibrationArgumentCollection recalArgs,
 
         covariates = new StandardCovariateList(recalArgs, readsHeader);
 
-        logger.info("The covariates being used here: ");
-        for (final Covariate cov : covariates) { // list all the covariates being used
-            logger.info('\t' + cov.getClass().getSimpleName());
-        }
-
```

Comment:
This affects the walker version too (which also uses the `BaseRecalibrationEngine`), so you should call the new utility from there as well.